Jenkinsfile.promote

```
@@ -159,7 +203,41 @@ void loginContainerRegistry(String registry, String credsId){
         sh "podman login -u ${REGISTRY_USER} -p ${REGISTRY_PWD} --tls-verify=false ${registry}"
     }
 }
+void readDeployProperties(){
+    if(params.DEPLOY_BUILD_URL != null){
+        sh "wget --auth-no-challenge --user=${JENKINS_USER} --password=${JENKINS_TOKEN} ${params.DEPLOY_BUILD_URL}artifact/deployment.properties"
+        deployProperties = readProperties file:"deployment.properties"
+        echo deployProperties.collect{ entry -> "${entry.key}=${entry.value}" }.join("\n")
+    }
+}
+boolean hasDeployProperty(String key) {
+    return deployProperties[key] != null
+}
+String getDeployProperty(String key) {
+    if(hasDeployProperty(key)){
+        return deployProperties[key]
+    }
+    return ""
+}
+String getParamOrDeployProperty(String paramKey, String deployPropertyKey) {
+    if (params[paramKey] != "") {
+        return params[paramKey]
+    }
+    return getDeployProperty(deployPropertyKey)
+}
+boolean isRelease() {
+    return params.RELEASE || (getDeployProperty("release") == "true")
+}
+String getProjectVersion() {
+    return getParamOrDeployProperty("PROJECT_VERSION" , "project.version")
+}
+String getGitTag() {
+    return params.GIT_TAG != "" ? params.GIT_TAG : getProjectVersion()
+}
 
+String getBuildBranch() {
+    return getParamOrDeployProperty("BUILD_BRANCH_NAME", "git.branch")
+}
```

Just add the `getGitAuthor` util method

```suggestion
String getBuildBranch() {
    return getParamOrDeployProperty("BUILD_BRANCH_NAME", "git.branch")
}

String getGitAuthor() {
    return getParamOrDeployProperty("GIT_AUTHOR", "git.author")
}
```